From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
Date: Sat, 01 Oct 2022 23:31:34 +0100
Subject: [PATCH] prevent ffmpeg version detection

---
 CMakeLists.txt | 4 +-
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 705b95f82..16c090c27 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -238,8 +238,8 @@ find_package(FFmpeg REQUIRED COMPONENTS AVCODEC AVFORMAT AVUTIL SWSCALE SWRESAMP
 if(FFmpeg_FOUND)
     SET(FFVER_OK TRUE)

-    # Can not detect FFmpeg version on Windows for now
-    if (NOT WIN32)
+    # Can not detect FFmpeg version on Windows/Android for now
+    if (NOT WIN32 AND NOT ANDROID)
         if(FFmpeg_AVFORMAT_VERSION VERSION_LESS "57.56.100")
             message(STATUS "libavformat is too old! (${FFmpeg_AVFORMAT_VERSION}, wanted 57.56.100)")
             set(FFVER_OK FALSE)
