diff --git a/apps/openmw/CMakeLists.txt b/apps/openmw/CMakeLists.txt
index 13fb71714..36f0540ab 100644
--- a/apps/openmw/CMakeLists.txt
+++ b/apps/openmw/CMakeLists.txt
@@ -185,7 +185,9 @@ if (CMAKE_VERSION VERSION_GREATER_EQUAL 3.16 AND MSVC)
 endif()

 if (ANDROID)
-    target_link_libraries(openmw EGL android log z)
+    find_package(OSGPlugins REQUIRED COMPONENTS ${USED_OSG_PLUGINS})
+    list(APPEND OPENSCENEGRAPH_LIBRARIES ${OSGPlugins_LIBRARIES})
+    target_link_libraries(openmw EGL android log z ${OPENSCENEGRAPH_LIBRARIES} freetype jpeg png)
 endif (ANDROID)

 if (USE_SYSTEM_TINYXML)
